testing/e2e/app/routes/Modal/spec.ts

```diff
@@ -0,0 +1,17 @@
+import { test, expect } from '@playwright/test';
+
+test('should not jump focus from a child to the dialog', async ({ page }) => {
```

Comment:
We should add a code comment linking to https://github.com/iTwin/iTwinUI/issues/2445 above this test.